ci/scripts/run_pytest.sh

```diff
@@ -25,13 +25,33 @@ if ! set_bionemo_home; then
     exit 1
 fi
 
-python -m coverage erase
+# download Codecov CLI
+curl -Os https://cli.codecov.io/latest/linux/codecov
 
+# integrity check
+curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import # One-time step
+curl -Os https://cli.codecov.io/latest/linux/codecov
+curl -Os https://cli.codecov.io/latest/linux/codecov.SHA256SUM
+curl -Os https://cli.codecov.io/latest/linux/codecov.SHA256SUM.sig
+gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
+
+shasum -a 256 -c codecov.SHA256SUM
+sudo chmod +x codecov
+
+# If pytests fail, we still want to ensure that the report is uploaded to codecov, so we set +e from here through the
+# upload.
+set +e
 for dir in docs/ ./sub-packages/bionemo-*/; do
     echo "Running pytest in $dir"
-    python -m coverage run --parallel-mode --source bionemo \
-        -m pytest -v --nbval-lax --durations=0 --durations-min=60.0 $dir
+    pytest -v --nbval-lax --cov=bionemo --cov-append --junitxml=$(basename $dir).junit.xml -o junit_family=legacy $dir
 done
 
-python -m coverage combine
-python -m coverage report --show-missing
+# Merge all sub-directory test results into a single xml file.
+junitparser merge *.junit.xml combined.junit.xml
+
+# Upload test analytics to codecov.
+./codecov do-upload --report-type test_results --file combined.junit.xml
```

Comment:
maybe I don't know how codecov works, but where is this being uploaded?

Reply:
Yeah IIUC this will upload by default to the standard codecov cloud, using our CODECOV_TOKEN to identify / authenticate